rpcs3/Emu/RSX/GSRender.cpp

```diff
@@ -20,19 +20,26 @@ GSRender::~GSRender()
 
 	if (m_frame)
 	{
-		m_frame->hide();
```

Comment:
Don't remove this, it's a workaround iirc. Due to the window handle being bound to a driver context and some drivers were buggy. Unfortunately nobody remembered to document exactly why, but at least it removes the game window from view in case the driver dies.

Reply:
^

Reply:
Okay, improved the workaround and moved to close() to avoid confusion.